<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Training Model</string>
    </property>
    <widget class="QLineEdit" name="processVariable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SI-04M1:CO-PT100-Ambient:Temp-Mon</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Process Variable (PV)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial Date</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="iniDatetime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>10</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>10</month>
       <day>9</day>
      </datetime>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Datetime</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="endDatetime">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>9</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>10</month>
       <day>11</day>
      </datetime>
     </property>
    </widget>
    <widget class="QPushButton" name="trainButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tags:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
